components/datatypes/marketing/commerce.schema.json

```diff
@@ -119,6 +119,11 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "A product has been purchased instantly, potentially skipping the cart or checkout.",
           "meta:descriptionId": "commerce##xdm:instantPurchase##description##9211"
+        },
+        "xdm:billing": {
```

Comment:
Billing Address is generally a function of the Customer. We already have a billing address on one of the profile based field groups ( https://github.com/adobe/xdm/blob/master/components/fieldgroups/profile/profile-personal-details.schema.json) . Shouldn't that suffice ?
We have other addresses there as well like, mailing/shipping etc.

Reply:
Sure. We can re use the billing object within personal details.